### ISSUE TITLE
0.8: not ready for sphinx 4.0.x?

### ISSUE THREAD

**Issue by kloczek** [repoze/repoze.sphinx.autointerface#18]
Trying to build documentation repoze-who I found that repoze.sphinx.autointerface is failing:
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.0.2
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing repoze.who.egg-info/PKG-INFO
writing dependency_links to repoze.who.egg-info/dependency_links.txt
writing entry points to repoze.who.egg-info/entry_points.txt
writing namespace_packages to repoze.who.egg-info/namespace_packages.txt
writing requirements to repoze.who.egg-info/requires.txt
writing top-level names to repoze.who.egg-info/top_level.txt
reading manifest file 'repoze.who.egg-info/SOURCES.txt'
writing manifest file 'repoze.who.egg-info/SOURCES.txt'
running build_ext
..............................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 302 tests in 0.052s

OK

Extension error:
Could not import extension repoze.sphinx.autointerface (exception: cannot import name 'force_decode' from 'sphinx.util' (/usr/lib/python3.8/site-packages/sphinx/util/__init__.py))
```

**Comment by dataflake** [comment]
Fixed by #17 , version 1.0.0 is now published on PyPI.

**Comment by kloczek** [comment]
Thank you 👍
